packages/vitest/src/node/config/schema.ts

```diff
@@ -0,0 +1,533 @@
+      thresholds: {
+        description: null,
+        argument: '', // no displayed
+        subcommands: {
+          perFile: {
+            description: 'Check thresholds per file. See --coverage.thresholds.lines, --coverage.thresholds.functions, --coverage.thresholds.branches and --coverage.thresholds.statements for the actual thresholds. This option is not available for custom providers (default: false)',
+          },
+          autoUpdate: {
+            description: 'Update threshold values: "lines", "functions", "branches" and "statements" to configuration file when current coverage is above the configured thresholds. This option is not available for custom providers (default: false)',
+          },
+          lines: {
+            description: 'Threshold for lines. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers',
+            argument: '<number>',
+          },
+          functions: {
+            description: 'Threshold for functions. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers',
+            argument: '<number>',
+          },
+          branches: {
+            description: 'Threshold for branches. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers',
+            argument: '<number>',
+          },
+          statements: {
+            description: 'Threshold for statements. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers',
+            argument: '<number>',
+          },
+          100: {
+            description: 'Shortcut to set all coverage thresholds to 100. This option is only available for the v8 provider (default: false)',
+          },
+        },
+      },
+      ignoreClassMethods: {
+        description: 'Array of class method names to ignore for coverage. Visit https://github.com/istanbuljs/nyc#ignoring-methods for more information. This option is only available for the istanbul providers (default: [])',
+        argument: '<name>',
+        array: true,
+      },
+      processingConcurrency: {
+        description: 'Concurrency limit used when processing the coverage results. (default min between 20 and the nubmer of CPUs)',
+        argument: '<number>',
+      },
+      customProviderModule: {
+        description: 'Specifies the module name or path for the custom coverage provider module. Visit https://vitest.dev/guide/coverage.html#custom-coverage-provider for more information. This option is only available for custom providers',
+        argument: '<path>',
+      },
+      // TODO: suport watermarks via a special command?
+      // CAC requires --watermarks.statements=50 --watermarks.statements=80 for "statements:[50,80]" which looks rediculous
+      watermarks: null,
```

Comment:
For now I think it's ok to not (publicly) support `coverage.watermarks` via CLI. It's not commonly used command and all cases I've seen have used configuration files.